ballerina/tests/tests.bal

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) 2024 WSO2 LLC. (http://www.wso2.org).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/log;
+import ballerina/os;
+import ballerina/test;
+
+configurable boolean isLiveServer = os:getEnv("IS_LIVE_SERVER") == "false";
```

Comment:
```suggestion
configurable boolean isLiveServer = os:getEnv("IS_LIVE_SERVER") == "true";
```

Reply:
@RDPerera the tests should be running against the mock service by default, right? Otherwise the tests will fail unless we set this to `false` via an env variable or the `Config.toml`

Reply:
Yes, 

This conditional statement `os:getEnv("IS_LIVE_SERVER") == "true"` will only be `true` when the IS_LIVE_SERVER environment variable is set to **TRUE**.

Therefore if we don't set that environment variable, the default test environment will be the mock server environment.

Reply:
Yes it make sense, Sorry for the confusion caused :) Updated